server/src/main/java/org/apache/iotdb/db/engine/compaction/cross/rewrite/task/RewriteCrossSpaceCompactionTask.java

```diff
@@ -134,23 +131,20 @@ private void executeCompaction()
             selectedSeqTsFileResourceList.get(0).getTsFile().getParent()
                 + File.separator
                 + targetTsfileResourceList.get(0).getTsFile().getName()
-                + PATH_SEPARATOR
-                + RewriteCrossSpaceCompactionLogger.COMPACTION_LOG_NAME);
+                + CompactionLogger.CROSS_COMPACTION_LOG_NAME_SUFFIX);
 
-    try (RewriteCrossSpaceCompactionLogger compactionLogger =
-        new RewriteCrossSpaceCompactionLogger(logFile)) {
+    try (CompactionLogger compactionLogger = new CompactionLogger(logFile)) {
       // print the path of the temporary file first for priority check during recovery
+      compactionLogger.logFiles(selectedSeqTsFileResourceList, STR_SOURCE_FILES);
+      compactionLogger.logFiles(selectedUnSeqTsFileResourceList, STR_SOURCE_FILES);
       compactionLogger.logFiles(targetTsfileResourceList, STR_TARGET_FILES);
-      compactionLogger.logFiles(selectedSeqTsFileResourceList, STR_SEQ_FILES);
-      compactionLogger.logFiles(selectedUnSeqTsFileResourceList, STR_UNSEQ_FILES);
       CompactionUtils.compact(
```

Comment:
may be we can close the logger before `compact`?

Reply:
Resolved